Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2628,3 +2628,50 @@ function Remove-CustomRole {
     }
 }
 
+<#
+    .Synopsis
+    This function allows the customer to restart HA on a cluster. This might be needed to clear HA alarms.
+    .EXAMPLE
+    Once the function is imported: Restart-ClusterHA -cluster_name "Cluster-1"
+#>
+function Restart-ClusterHA {
+    [AVSAttribute(30, UpdatesSDDC = $false)]
```

Comment:
@Zsoldier I'm wondering if it would be appropriate to lock the SDDC for the duration of this commandlet (`UpdatesSDDC` parameter).

Reply:
Agreed

Reply:
Would this be able to run if SDDC is locked?

Reply:
No Run Command can execute when SDDC is locked (in anything but Fulfilled state). This parameter controls w/ starting this commandlet should put in Updating state.